Fixes #11024 - Add channel info to snap installer docs (#11079)
MicrosoftDocs/PowerShell-Docs: reference/docs-conceptual/install/install-other-linux.md

pwsh
```

> [!NOTE]

> > Information the user should notice even if skimmingMicrosoft only supports the `latest/stable` and
> > `lts/stable` channels for the `powershell` package. Do not install packages from the other
> > channels.

Reviewer: ```suggestion
> Microsoft only supports the `latest/stable` and `lts/stable` channels for the `powershell`
> package. Do not install packages from the other channels.
```

pwsh-preview
```

> [!NOTE]

> > Information the user should notice even if skimmingMicrosoft only supports the `latest/stable`
> > channel for the `powershell-preview` package. Do not install packages from the other channels.

Reviewer: ```suggestion
> Microsoft only supports the `latest/stable` channel for the `powershell-preview` package. Do not
> install packages from the other channels.
```
